Team_Roster/app/templates/base.html

```diff
@@ -68,4 +64,10 @@ <h4> MLB Player Roster - Team Immaculate Party Time</h4>
     </div>
 </body>
 
+        <meta charset="UTF-8">
```

Comment:
why is this added here? It is already in the code above in the `<head>`, where this content is meant to be, I think you can remove this.